src/hooks/tradingView/useTradingView.ts

```diff
@@ -63,6 +90,18 @@ export const useTradingView = ({
       tvWidgetRef.current = tvChartWidget;
 
       tvWidgetRef.current.onChartReady(() => {
+        tvWidgetRef?.current?.headerReady().then(() => {
+          const button = tvWidgetRef?.current?.createButton();
+
+          if (button) {
+            button.innerHTML = `<span>${stringGetter({
+              key: STRING_KEYS.ORDER_LINES,
+            })}</span> <div class="displayOrdersButton-toggle"></div>`;
```

Comment:
😭 easiest way I could think of to make a toggle in pure css (instead of doing a toggle via svg, which made themed colors really hard) but open to other suggestions (because this is not my fav code)